vmm/rpc/proto/vmm_rpc.proto

```diff
@@ -224,6 +224,41 @@ message GpuInfo {
   bool is_free = 4;
 }
 
+message BackupDiskRequest {
+  // vm id
+  string vm_id = 1;
+  // full or incremental
+  string level = 2;
```

Comment:
Is it better to use Enum for readability?

Reply:
It's lack of syntax in the proto to express `#[serde(rename = "snake_case")]` when we convert it to json API.
So, I usually avoid to use enum.